recipes/phasefieldx/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "phasefieldx" %}
+{% set version = "0.0.3" %}
```

Comment:
There is a 0.0.4 on PyPI. Is there a reason to not use the source there?

Reply:
Thanks for pointing that out! There was no specific reason for not using version 0.0.3. I’ve just updated the project to version 0.0.4    https://github.com/conda-forge/staged-recipes/pull/27848/commits/a5096ea59bf373dc04ed717a5ea5182433c57c54